src/components/Slider/Slider.jsx

```diff
@@ -0,0 +1,96 @@
+import React from "react";
+import { Swiper, SwiperSlide } from "swiper/react";
+import { Pagination, Navigation } from "swiper/modules";
+import SwiperCore from "swiper";
+import "./Slider.css";
+
+import "swiper/css";
+import "swiper/css/bundle";
+
+SwiperCore.use([Navigation, Pagination]);
+
+const Slider = ({ records }) => {
+  const [windowWidth, setWindowWidth] = React.useState(window.innerWidth);
+
+  React.useEffect(() => {
```

Comment:
Import useState and useffect so you don't have to refrences react: 

import React { useState, useEffect } from "react";

useState(window.innerWidth)

Reply:
completed